README.md

```diff
@@ -89,17 +89,22 @@ If you would like to tail the logs in the console then you remove the detach fla
 
 The following are examples of some common Django management commands that you may need to run.
 
-* Make Migrations: `docker-compose run --rm manage makemigrations`
-* Merge Migrations: `docker-compose run --rm manage makemigrations --merge`
-* Run Migrations: `docker-compose run --rm manage`
-* Test: `docker-compose run --rm manage test`
+* Make Migrations: `docker-compose run --rm app ./manage.py makemigrations`
+* Merge Migrations: `docker-compose run --rm app ./manage.py makemigrations --merge`
+* Run Migrations: `docker-compose run --rm app ./manage.py`
```

Comment:
I think this is supposed to be `docker-compose run --rm app ./manage.py migrate` (missing the "migrate" :))

Reply:
:+1: Good call fixed in 98b567a

Reply:
Thanks!